src/sass/shared/variables.scss

```diff
@@ -85,11 +91,21 @@ $font-size-title-xsmall: 21px;
 $font-size-small: 18px;
 $font-size-xsmall: 16px;
 $font-size-xxsmall: 14px;
+$font-size-micro: 12px;
 $font-size-footer-titles: 21px;
 $font-size-footer-text: $font-size-xsmall;
 $font-size-secondary-nav-item: 21px;
 $font-size-testimonial-usernames: 18px;
 
+// Line Height
+$line-height-condensed: 1em;
+$line-height-small: 1.2em;
+$line-height-projects-title: 30px;
+
+// Font weight
+$font-weight-bold: bold;
```

Comment:
We moved to `shared/variables.scss` many values that are not going to change for sure.
I just extracted it because it is a typography property and the rest of typography properties are already defined in this file. I can use the bold value if you prefer.